mindsdb/migrations/versions/2023-02-17_ee63d868fa84_predictor_integration_null.py

```diff
@@ -0,0 +1,42 @@
+"""predictor_integration_null
+
+Revision ID: ee63d868fa84
+Revises: b6d0a47294ac
+Create Date: 2023-02-17 13:48:49.464644
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import mindsdb.interfaces.storage.db # noqa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ee63d868fa84'
+down_revision = 'b6d0a47294ac'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    with op.batch_alter_table('predictor', schema=None) as batch_op:
```

Comment:
should be there also migration for jobs.project_id?

Reply:
It is already 'nullable=False' in previous migration.